enigma/src/main/java/cuchaz/enigma/translation/representation/entry/ClassEntry.java

```diff
@@ -238,15 +238,6 @@ public static String getInnerName(String name) {
 		return name;
 	}
 
-	@Override
-	public String getSourceRemapName() {
```

Comment:
Correct me if I'm wrong, but the default implementation of `getSourceRemapName` returns `this.getName()` right? And `getName` returns the fully qualified name for non-inner classes.

Reply:
oh yes that's true, I was looking at the wrong thing when I determined it could be entirely removed